src/Widgets/NotificationEntry.vala

```diff
@@ -17,12 +17,16 @@
 
 public class Notifications.NotificationEntry : Gtk.ListBoxRow {
     public signal void clear ();
+    public signal void removeNotificationEntry (NotificationEntry entry);
```

Comment:
We use `remove_notification_entry` naming convention in signals as well. It is also not needed to explicitly declare the entry parameter in the signal itself, Vala will by default pass it to you if there are no arguments (I believe).
```suggestion
    public signal void remove_notification_entry ();
```